torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -612,7 +612,7 @@ const std::vector<std::string> functions = {
             size = len(tensors)
             split_sizes = [0] * size
             for i in range(size):
-                if tensors[i].numel() > 0:
+                if tensors[i].numel() > 0 or len(tensors[i].size()) != 1:
```

Comment:
i guess this is the equivalent logic : https://codebrowser.bddppq.com/pytorch/pytorch/torch/csrc/autograd/FunctionsManual.cpp.html#602

expressed in python as tensors[i].size() != [0]. should we do that here as well ?